apps/src/applab/commands.js

```diff
@@ -1008,12 +1010,12 @@ applabCommands.dropdown = function(opts) {
   }
   newSelect.id = opts.elementId;
   newSelect.style.position = 'relative';
-  newSelect.style.fontSize = defaultFontSizeStyle;
-  newSelect.style.fontFamily = fontFamilyStyles[0];
   if (experiments.isEnabled('applabThemes')) {
     newSelect.style.borderStyle = 'solid';
     elementLibrary.applyCurrentTheme(newSelect, Applab.activeScreen());
   } else {
+    newSelect.style.fontSize = defaultFontSizeStyle;
+    newSelect.style.fontFamily = fontFamilyStyles[0];
```

Comment:
Echoing @islemaster's feedback on the original PR, can you unify this logic for the experiment and non-experiment case? https://github.com/code-dot-org/code-dot-org/pull/27644#discussion_r269662193

Unless you _really_ don't think we'll go this direction for shipping the code.

Reply:
The goal is to flip on the experiment and delete the old code within the next 1-2 weeks. I'd rather preserve the existing code paths as much as possible for the non-experiment case until I delete it at the end of the month.